src/markdown-pages/docs/cli-command-reference/runs-get.md

```diff
@@ -3,30 +3,146 @@ path: "/docs/cli-command-reference/cli-runs-get"
+
+
+Table 2: options that you can set on the `galasactl runs get` command to display test run results in different formats:
+
+| Name |  Description  |
+| :---- | :-------- | 
+| `--format summary` | The default format is _summary_. The _summary_ format is useful if you just need to know the high-level status and result of a test.  | 
+| `--format details` | The _details_ format is useful when you need to see all the details of a test run, including method information.  | 
+| `--format raw` | The _raw_ format output is useful if you are writing scripts to report on multiple test runs programmatically. The output from `galasactl runs get` is returned in a form that makes it easy for scripting to digest the individual pieces of data available, without having to query the data via the published REST interface. | 
+
+
+## View tests results in summary format
+
+The following example command returns test status in a summary format:
+
+```
+galasactl runs get --name U456 --bootstrap http://example.com:30960/boostrap 
+``` 
+
+where:
+- `--name` is the name of the test run 
+- `--bootstrap` is the URL of the ecosystem's bootstrap properties
+
+
+
+Results are returned on the terminal in the following example format:
+
+```
+$galasactl runs get --name U456 --format summary
+submitted-time      name status   result             test-name
+2023-05-04 10:55:29 U456 Finished MyLongResultString MyTestName
+```
+
+## View tests results in details format
+
+The following example command returns test status in a details format:
+
+On Mac or Unix:
+
+```
+$galasactl runs get --name U456 --bootstrap http://example.com:30960/boostrap \
+--format details 
+``` 
+
+On Windows (Powershell):
 ```
-galasactl runs get --runname <NameOfTestRun> --bootstrap <BootstrapURL>
+galasactl runs get --name U456 --bootstrap http://example.com:30960/boostrap `
+--format details 
 ``` 
 
 where:
-- `--runname` is the name of the test run 
+- `--name` is the name of the test run 
 - `--bootstrap` is the URL of the ecosystem's bootstrap properties
+- `--format` is the format in which you want the results to display
 
 
 Results are returned on the terminal in the following example format:
 
 ```
-galasactl runs get --runname U456 
-RunName Status   Result ShortTestName
-U456    Finished Passed MyTestName
+galasactl runs get --name U456 --format details
+name            :  U456
+status          :  Finished
+result          :  Passed
+submitted-time  :  2023-05-04 10:55:29
+start-time      :  2023-05-05 06:00:14
+end-time        :  2023-05-05 06:00:15
+duration(ms)    :  1000
+test-name       :  dev.galasa.Zos3270LocalJava11Ubuntu
+requestor       :  galasa
+bundle          :  dev.galasa
+run-log         :  https://127.0.0.1/ras/run/cbd-123/runlog
+
```

Comment:
the spacing between the colons have chnaged slightly - there should now only be one space after the colon
for example: 'submitted-time : 2023-05-04 10:55:29'